### DISCUSSION TITLE
How to add logger/loggerFactory to OpenAIAssistantAgent

### DISCUSSION THREAD

**Discussion by JP-droidic** [microsoft/semantic-kernel#8619] [Q&A] [answered]
Hi team

can you please provide an example of adding logger/loggerFactory  to an OpenAIAssistantAgent instance.

I saw here https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/GettingStartedWithAgents/Step07_Logging.cs example for ChatCompletionAgent but can't find example for OpenAIAssistantAgent

thanks

**Comment by evchaki** [comment, marked as answer]
@JP-droidic - you should be able to use the link you provided above to log on the OpenAI Assistant API (Agent).  Let me know if you are stuck.

**Reply by JP-droidic** [reply]
thanks @evchaki, I completely missed the line

cheers
